client/src/templates/Guide/GuideArticle.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { graphql } from 'gatsby';
 
 import ArticleLayout from './components/ArticleLayout';
+import GuideFooter from './components/GuideFooter';
```

Comment:
How about posting this in an article layout?

Reply:
I wasn't exactly sure what you meant here, I took it as moving the `GuideFooter` component to the `ArticleLayout` component. So I did that, let me know if it's not what you wanted.